StripeCryptoOnramp/StripeCryptoOnramp/Source/Components/CryptoOnrampCoordinator.swift

```diff
@@ -53,8 +56,9 @@ public protocol CryptoOnrampCoordinatorProtocol {
 
     /// Creates an identity verification session and launches the verification flow.
     ///
+    /// - Parameter viewController: The view controller from which to present the verification flow.
     /// - Returns: An `IdentityVerificationResult` representing the outcome of the verification process.
-    func promptForIdentityVerification() async throws -> IdentityVerificationResult
+    func promptForIdentityVerification(from viewController: UIViewController) async throws -> IdentityVerificationResult
```

Comment:
Just highlighting here that this is a change to the `public` API in case the mobile API review doc needs to be updated.